### DISCUSSION TITLE
no plane showing in moving map

### DISCUSSION THREAD

**Discussion by ajts1958** [flybywiresim/aircraft#4603] [Help] [answered]
Since the FBW A320Neo became a stand alone aircraft, both in experimental and dev versions I have no airplane in flypad moving map. 
Can anyone tell me why this is hapenning?

**Comment by ExampleWasTaken** [comment, marked as answer]
Please uninstall the aircraft from your community folder and reinstall it through the [installer](https://api.flybywiresim.com/installer). 

**NOTE:** A bug, introduced in a recent sim update, causes the aircraft on the EFB to flicker. This is not something we can fix.

**Reply by ajts1958** [reply]
I uninstalled the previous mod and installed the stand alone with the installer
Thank you very much for your reply
It's not very important
Best regards

**Reply by ExampleWasTaken** [reply]
When did you install it?

**Reply by ajts1958** [reply]
one week after the stand alone version launch

**Reply by ExampleWasTaken** [reply]
update to the latest version.
